proof/refine/ARM/StateRelation.thy

```diff
@@ -192,13 +196,20 @@ where
   \<and> tcb_bound_notification tcb = tcbBoundNotification tcb'
   \<and> tcb_mcpriority tcb = tcbMCP tcb'"
 
+\<comment> \<open>
+  A pair of objects @{term "(obj, obj')"} should satisfy the follow relation when, under further
```

Comment:
```suggestion
  A pair of objects @{term "(obj, obj')"} should satisfy the following relation when, under further
```
Other architectures as well.

Reply:
And thanks for the comment, it will be helpful if/when in the future we're confused about why it doesn't handle TCBs.